### ISSUE TITLE
Weird error message from BGA

### ISSUE THREAD

**Issue by DoctorMikeReddy** [DavidEGx/Hive-bga2bs#11]
When you open a game on BGA to replay, then hit the javascript bookmark (as per instructions) a popup error message appears:
![image](https://user-images.githubusercontent.com/1001087/80966486-80361080-8e0c-11ea-90b4-ff7a803ec5de.png)
Clicking OK, the .sgf file still downloads, but the Popup keeps reappearing, and the whole tab/page needs to be closed

**Comment by DoctorMikeReddy** [comment]
Clicking on another tab - changing pages - seems to clear the popup and download then commences, with the page defaulting back to the "Choose Player" screen. So, this isn't exactly a bug...

**Comment by DoctorMikeReddy** [comment]
Sorry, to clarify further:
Using Chrome on Windows 10 (can check on iOS and MacOS, and other browsers if you need me too)
All my archived games cause this popup to appear, but games by others too, it seems; I clicked on a player I'd played, but a game I wasn't a part of, and the same thing happened

**Comment by DavidEGx** [comment]
What are describing is a bit weird, cannot really reproduce.

This is what currently should happen:

**Case 1**
You are on the page `/gamereview?table=NUMBER`
(you haven't clicked "choose this player")

Download should work.


**Case 2**
You are on the page `/archive/replay/NUMBER/?table=NUMBER&player=NUMBER`
(you clicked "choose this player")
and the player you selected has "Tournament" notation style selected.

Download should work.


**Case 3**
You are on the page `/archive/replay/NUMBER/?table=NUMBER&player=NUMBER`
(you clicked "choose this player")
and the player you selected has NOT "Tournament" notation style selected.

Download won't work. You'll get the popup and will be redirected so you are in case 1.

---

If the player you are selecting via "choose this player" is yourself, I suggest you go to options and change the notation style to Tournament.

---

Nonetheless the popup could be removed I think. I'll have a look.

**Comment by DoctorMikeReddy** [comment]
Shall I send you what it has downloaded?

**Comment by DoctorMikeReddy** [comment]
Sorry, how do I change preferences to use Tournament notation style?

**Comment by DoctorMikeReddy** [comment]
Right, I worked out how to change to Tournament notation, and the error didn’t appear (Yey!) and got this file (attached). Is the “error” just cosmetic? I’ll try BS review now, to see if any of them are problematic

**Comment by DoctorMikeReddy** [comment]
Hoping you aren’t sick of me now. The Tournament notation version (a game I am currently playing) worked. The ones before I knew I had to choose notation style do not, but DO download. So, we are in case 3 above, I think. Weirdly, in a current game, the script told me to choose tournament notation before it would work (in a Popup). Can you change the notation in a finished game? Or set this as the default when you start a new one?

**Comment by DoctorMikeReddy** [comment]
Ok, just to be diligent, it looks like you cannot retrospectively change notation on a finished game. Something useful to maybe put in the readme for the script instructions. Weird, as the files look very similar, but obviously there is something “wrong” in the classic notation versions that the offline reviewer from BoardSpace cannot read?

**Comment by DoctorMikeReddy** [comment]
OK! Playing with this, if you use the script BEFORE the Choose Player bit, but after the Reply Game bit, you can see tournament notation in the game log, and the script works! Only if you have chosen a player does the option for tournament notation not stick, and the script fails. So, not sure if I didn't follow instructions, but it works now, even for games that didn't originally use tournnament notation. This is not the way the instructions tell me to use the script, as the animation shows I have to choose a player before running it; this does work for games that WERE set to tournament notation, such as the current running games that I could change this for

tl;dr might be worth testing and/or changing the instructions, to either not choose player, or make sure notation is set properly

**Comment by DavidEGx** [comment]
I've updated gif and instructions to avoid confusion.

Leaving this issue open though, I think it is fixable so I'll fix it ( at some point :) )

**Comment by DoctorMikeReddy** [comment]
Brilliant, thanks
